fetch/cross-origin-resource-policy/fetch.any.js

```diff
@@ -40,7 +40,7 @@ promise_test((test) => {
 
 promise_test((test) => {
     const remoteURL = httpsBaseURL + "resources/hello.py?corp=same-site";
-    return fetch(remoteURL, { mode: "no-cors" });
+    return promise_rejects(test, new TypeError, fetch(remoteURL, { mode: "no-cors" }));
 }, "Cross-origin no-cors fetch to a same-site URL with a 'Cross-Origin-Resource-Policy: same-site' response header.");
```

Comment:
Please make this say "cross-scheme same-site" or some such. I forgot to read the PR description and it took me a long time to figure out what was going on.

Reply:
OK, added Cross-scheme (HTTP->HTTPS)